contributors/devel/godep.md

```diff
@@ -90,6 +90,7 @@ rm -rf vendor
 ./hack/update-bazel.sh
 ./hack/update-godep-licenses.sh
 ./hack/update-staging-client-go.sh
+./hack/update-staging-godeps.sh
```

Comment:
The order is correct. Also it's worth mentioning one needs to make sure before running this script, there should be no client-go or other staging repo in the GOPATH.